dask/array/tests/test_array_core.py

```diff
@@ -5103,6 +5104,7 @@ def _known(num=50):
 def unknown():
     x = _known()
     y = x[x < 0]
+    y._chunks = ((np.nan,) * 5,)
     assert y.chunks == ((np.nan,) * 5,)
     return y
```

Comment:
The original intent of the code was to organically create an array with unknown chunk sizes by using the boolean filter. Since we are now forcing the chunks to be unknown, the slice only adds confusion. So I think this should be:

```suggestion
    x._chunks = ((np.nan,) * 5,)
    return x
```